RELEASING.md

```diff
@@ -8,7 +8,7 @@
 1. Confirm that the kotlin build is green before committing any changes
    (Note we exclude benchmarks, but you can check those too!)
    ```bash
-   ./gradlew build connectedCheck ./gradlew connectedCheck -x :benchmarks:dungeon-benchmark:connectedCheck
+   ./gradlew build connectedCheck ./gradlew connectedCheck -x :benchmarks:dungeon-benchmark:connectedCheck -x :benchmarks:performance-poetry:complex-benchmark:connectedCheck
```

Comment:
I just pushed a conflicting change b/c this is an invalid command line, and has a redundant bit:
```suggestion
   ./gradlew build && ./gradlew connectedCheck -x :benchmarks:dungeon-benchmark:connectedCheck -x :benchmarks:performance-poetry:complex-benchmark:connectedCheck
```

Reply:
Whoops thank you